test/src/api/routes/test-author.js

```diff
@@ -156,3 +165,117 @@ describe('GET /Author', () => {
+		};
+		await new RelationshipType(relationshipTypeData)
+			.save(null, {method: 'insert'});
+
+		const relationshipsPromise = [];
+		for (const authorBBID of authorBBIDs) {
+			const relationshipData = {
+				id: random.number(),
+				sourceBbid: authorBBID,
+				targetBbid: work.get('bbid'),
+				typeId: relationshipTypeData.id
+			};
+			relationshipsPromise.push(
+				new Relationship(relationshipData)
+					.save(null, {method: 'insert'})
+			);
+		}
+		const relationships = await Promise.all(relationshipsPromise);
+
+		const workRelationshipSet = await new RelationshipSet({id: random.number()})
+			.save(null, {method: 'insert'})
+			.then((model) => model.relationships().attach(relationships).then(() => model));
+
+		work.set('relationshipSetId', workRelationshipSet.get('id'));
+		work.set('revisionId', revision.get('id'));
+		await work.save(null, {method: 'update'});
+	});
+	after(truncateEntities);
+
+
+	it('should return list of authors associated with the work (without any filter)', async () => {
+		const res = await chai.request(app).get(`/author?work=${work.get('bbid')}`);
+		await browseAuthorBasicTests(res);
+		expect(res.body.authors.length).to.equal(3);
+	});
+
+	it('should return list of authors associated with the work (with Type filter)', async () => {
+		const res = await chai.request(app).get(`/author?work=${work.get('bbid')}&type=Author+Type+1`);
+		await browseAuthorBasicTests(res);
+		expect(res.body.authors.length).to.equal(1);
+		expect(_.toLower(res.body.authors[0].entity.authorType)).to.equal('author type 1');
+	});
+
+	it('should return 0 authors (with Incorrect Type filter)', async () => {
+		const res = await chai.request(app).get(`/author?work=${work.get('bbid')}&type=wrongFilter`);
+		await browseAuthorBasicTests(res);
+		expect(res.body.authors.length).to.equal(0);
+	});
+
+	it('should allow params to be case insensitive', async () => {
+		const res = await chai.request(app).get(`/aUThor?wOrk=${work.get('bbid')}&tYPe=AuTHor+TyPe+1`);
+		await browseAuthorBasicTests(res);
+		expect(res.body.authors.length).to.equal(1);
+		expect(_.toLower(res.body.authors[0].entity.authorType)).to.equal('author type 1');
+	});
+
+	it('should NOT throw an error if there is no related entity', async () => {
+		const work2 = await createWork();
+		const res = await chai.request(app).get(`/author?work=${work2.get('bbid')}`);
+		await browseAuthorBasicTests(res);
+		expect(res.body.authors.length).to.equal(0);
+	});
+
+	it('should throw 400 error for invalid bbid', (done) => {
+		chai.request(app)
+			.get('/author?work=1212121')
+			.end(function (err, res) {
+				if (err) { return done(err); }
+				expect(res).to.have.status(400);
+				return done();
+			});
+	});
+
+	it('should throw 404 error for incorrect bbid', (done) => {
+		chai.request(app)
+			.get(`/author?work=${aBBID}`)
+			.end(function (err, res) {
+				if (err) { return done(err); }
+				expect(res).to.have.status(404);
+				return done();
+			});
+	});
+});
```

Comment:
You could also add a test to check the a 400 error is returned if  passing more than one type parameter, for all entity types expect Edition Group